packages/mux-uploader-react/README.md

```diff
@@ -89,8 +89,10 @@ const MuxUploaderWithMuxUploaderDropExample = () => {
 
 `<MuxUploader>` has a handful of events to monitor uploading state.
 
-| Prop         | Description                                                                |
-| ------------ | -------------------------------------------------------------------------- |
-| `onError`    | Invoked when an error occurs in the chunked upload process.                |
-| `onProgress` | Invoked whenever a chunk of the file has successfully completed uploading. |
-| `onSuccess`  | Invoked when the entire file has successfully completed uploading.         |
+| Prop             | Description                                                                |
+| ---------------- | -------------------------------------------------------------------------- |
+| `onAttempt`      | Invoked immediately before a chunk upload is attempted.                    |
+| `onChunkSuccess` | Invoked when an indvidual chunk is successfully uploaded.                  |
+| `onError`        | Invoked when an error occurs in the chunked upload process.                |
+| `onProgress`     | Invoked whenever a chunk of the file has successfully completed uploading. |
```

Comment:
nitpick(non-blocker): I can't tell the difference between `onProgress` and `onChunkSuccess` based on these descriptions (and legit don't know the difference in upchunk). Consider rewording to make the difference clearer.